openapi/media_type.go

```diff
@@ -8,20 +8,31 @@ type MediaType struct {
 	Schema *Schema
 }
 
-func (m *MediaType) GetName() string {
-	return m.parent.GetName() + m.name
-}
+//func (m *MediaType) GetName() string {
+// TODO where is this used?
+//
+//	return m.parent.GetName() + "[" + m.node.name + "]"
+//}
 
-func (m *MediaType) getRef() string {
-	return ""
-}
+//func (m *MediaType) getRef() string {
+// FIXME node composable should override this
+//if m.Schema != nil {
+//	return m.Schema.Ref
+//}
+//fmt.Println("borked in media type getref")
+//return ""
+//}
```

Comment:
Pretty happy for this to waai. Please remove it entirely instead of leaving the commented code.